docs/get_started.md

```diff
@@ -98,81 +100,64 @@ the creation of a new management pack project.
 
-8. `Select a template for your project`
+8. `Select a language for the adapter:`
 
-     Both of the available options will generate a project structure that can be modified into an adapter:
+    The language the adapter uses determines the initial project structure.
+    Along with additional language-specific configuration.
+    Currently, `mp-init` offers the following languages:
 
-     `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
-      The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+    `Python`
+ 
+    `Java` 
 
-     `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
-      connection, collection, adapter definition, and endpoints logic.
+9. `(For Java projects only) Enter package name: `
 
-     For the purposes of this *Get Started* guide, select **Sample Adapter**.
+    The package name will be used to set up the package used by the adapter and the directory structure of the project.
+   
+    ???+ example 
+ 
+        ``` java
+        package com.mycompany;
+        ```
+
+10. `Select a template for your project`
+
+    Both of the available options will generate a project structure that can be modified into an adapter:
+
+    `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
+    The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+
+    `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
+    connection, collection, adapter definition, and endpoints logic.
+
+    For the purposes of this *Get Started* guide, select **Sample Adapter**.
 
 ???+ info
 
      For complete documentation of the `mp-init` tool, including an overview of its output,
-     see the [MP Initialization Tool Documentation](references/mp-init.md).
+     see the [MP Initialization Tool Documentation](references/mp-init.md). 
+     For Java projects documentation consult [Java project](references/python_project/index.md) section   
+     For Python projects documentation consult [Python project](references/java_project/index.md) section   
 
 ### Project Templates
-Both templates create a file system that has the basic project structure required to develop and build a Management Pack.
-Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation. `app/adapter.py` is the adapter's
-entry point and the best starting point.
+All the project templates create a file system that has the basic project structure required to develop and build a Management Pack.
+Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation.
```

Comment:
We just linked to `mp-init` documentation a few lines ago. Let's remove one of them. Actually, this whole `Project Templates` section is feeling a bit out of place. We definitely want the information here, but maybe not in the 'get started' page.

We talked a bit about having a "Next Steps" section at the conclusion of the walkthrough. I think we can put a list of documents there that will give some guidance about how to start creating their own adapter without interrupting the flow of doing this walkthrough.